src/NumberInput/index.d.ts

```diff
@@ -1,9 +1,11 @@
 import * as React from 'react';
 
+import { OmitPolyfill } from '../common';
 import {InputProps} from '../Input';
 
-export interface NumberInputProps extends InputProps {
+export type NumberInputProps = OmitPolyfill<InputProps, 'onChange'> & {
   strict?: boolean;
+  onChange?: (value: number) => void;
```

Comment:
```suggestion
  onChange?: (value: number | null) => void;
```
https://github.com/wix/wix-style-react/blob/62bc64a783678060db52a3657bf9165a4d12c991/src/NumberInput/NumberInput.js#L75